cmd/koord-runtime-proxy/main.go

```diff
@@ -64,5 +64,5 @@ func main() {
 
 	stopCh := genericapiserver.SetupSignalHandler()
 	<-stopCh
-	klog.Info("koordiantor runtime-proxy shutting down")
+	klog.V(1).Info("koordiantor runtime-proxy shutting down")
```

Comment:
Compared with V1, the direct Info is more concise. It seems that the distinction between the two is not that big.